src/platform/Ameba/BLEManagerImpl.h

```diff
@@ -156,15 +148,27 @@ class BLEManagerImpl final : public BLEManager, private BleLayer, private BlePla
     CHIP_ERROR SetSubscribed(uint16_t conId);
     bool UnsetSubscribed(uint16_t conId);
     bool IsSubscribed(uint16_t conId);
-
+    void InitSubscribed(void);
     bool RemoveConnection(uint8_t connectionHandle);
     void AddConnection(uint8_t connectionHandle);
 
     BLEManagerImpl::CHIPoBLEConState * GetConnectionState(uint8_t connectionHandle, bool allocate);
+#if defined(CONFIG_MATTER_BLEMGR_ADAPTER) && CONFIG_MATTER_BLEMGR_ADAPTER
+    static CHIP_ERROR matter_blemgr_gap_connect_cb(uint8_t conn_id);
+    static CHIP_ERROR matter_blemgr_gap_disconnect_cb(uint8_t conn_id, uint16_t disc_cause);
+    static void matter_blemgr_rx_char_write_cb(uint8_t conn_id, uint8_t * p_value, uint16_t len);
+    static void matter_blemgr_tx_char_cccd_write_cb(uint8_t conn_id, uint8_t indicationsEnabled, uint8_t notificationsEnabled);
+    static CHIP_ERROR matter_blemgr_tx_complete_cb(uint8_t conn_id);
+    static int matter_blemgr_callback_dispatcher(void * param, T_MATTER_BLEMGR_CALLBACK_TYPE cb_type, void * p_cb_data);
+#else
+#if CHIP_ENABLE_ADDITIONAL_DATA_ADVERTISING
```

Comment:
add a comment why additional data advertisement does not work or is not needed with matter blemgr adapter

Reply:
Fixed, it should also work with matter_blemgr_adapter.